autoblocks/_impl/testing/models.py

```diff
@@ -71,7 +79,7 @@ def hash(self) -> str:
         return self._cached_hash
 
 
-class BaseTestEvaluator(abc.ABC):
+class BaseTestEvaluator(abc.ABC, Generic[TestCaseType, OutputType]):
```

Comment:
evaluators need generics for test case and output, so this now matches how it's set up in TS